lunatrace/bsl/hasura/migrations/lunatrace/1679066158985_add_ml_tables/up.sql

```diff
@@ -0,0 +1,22 @@
+
+CREATE TABLE vulnerability.code_snippet
+(
+    id          uuid                        DEFAULT public.gen_random_uuid() NOT NULL PRIMARY KEY,
+    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    -- Reference may be null because we may have pulled code from a non-web source such as vuln-db
+    reference_id uuid NULL references vulnerability.reference,
+    -- Include url since reference might be null but its still nice to be able to point a source like a vuln-db link for non-scraped content
+    source_url text NOT NULL,
+    vulnerability uuid NOT NULL references vulnerability.vulnerability,
+    code text NOT NULL,
+    score integer NOT NULL,
+    summary text NOT NULL,
+    type text NOT NULL,
+    language text NOT NULL
+);
+
+-- Add a generated summary to the reference to make it easier for the LLM to choose what to read
+ALTER TABLE vulnerability.reference_content ADD COLUMN summary text NULL;
```

Comment:
what is the difference between this column and https://github.com/lunasec-io/lunasec/blob/master/lunatrace/bsl/hasura/migrations/lunatrace/1678406466712_add_parsed_content_to_reference_content/up.sql#LL1C81-L1C81?

Reply:
We also discussed this the other day, this is a short one to two sentence description of the content, so that we can display it in a list for the LLM to choose what to read.